api/graphql.go

```diff
@@ -0,0 +1,109 @@
+/*
+Gets factors that are used to build the rating for correctness of modules via
+GitHub's GraphQL API. getCorrectnessFactors function returns factors in the
+order of watchers, stargazers, totalCommits. A data type in the same form as
+the query structure is required to convert string to json. From json, the data
+is returned.
+
+--Use of GitHub token needs to be changed
+*/
+
+package api
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+)
+
+type CorrectnessFactors struct {
+	Data struct {
+		Repository struct {
+			StargazerCount int64
+			Watchers       struct {
```

Comment:
what is this spacing

Reply:
I tried following the structure of the query I made on Github GraphQL explorer. Were you referring to the int64 spacing or all the structs?

Reply:
the watchers one specifically, its just weird